docs/kubernetes/ingress-usage.md

```diff
@@ -79,9 +79,9 @@ Cloud migration.
 ### Multiple Ingresses defining the same route
 
 !!! Warning
-    
+
```

Comment:
Not sure if this will render correctly

Reply:
This works fine when I run it locally with `mkdocs serve -s`

![image](https://user-images.githubusercontent.com/128566/52936758-7508d900-335d-11e9-8a90-665732f0c9bc.png)